doc/gallery/introduction/pytensor_intro.ipynb

```diff
@@ -0,0 +1,3407 @@
+{
```

Comment:
<p>JAX comes a bit out of the blue here, we weren't talking about it before (were we)? You can keep the numba part. JAX doesn't really overload numpy anymore (although it might again with the dispatch thing that's going on in numpy but it's not yet advertised)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pytensor/pull/1585/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>removed JAX part</p>

Reply:
<blockquote>Furthermore, these manipulations are done within Python. JAX, for example, is able to trace the above function to construct a computational graph. But the graph representation is not front-and-center, and it's no obvious how to use it. In PyTensor, you are meant to be working direcly on graphs!</blockquote><p>I would remove this part?</p>

Reply:
Removed!